src/App.js

```diff
@@ -31,9 +31,12 @@ function App() {
           <Routes>
             <Route path="/" element={<Home />} />
             <Route path="/articles" element={<Articles />} />
-            <Route path="/topics" element={<Topics />} />
             <Route path="/articles/:topic" element={<Articles />} />
             <Route path="/article/:article_id" element={<ArticleCard />} />
+            <Route
+              path="/article/:article_id/comments"
+              element={<ArticleComments />}
+            />
```

Comment:
I don't think you ever use this route as you render `ArticleComments` inside of `ArticleCard`. You can get rid of this